Audiobook Player/PlayerManager.swift

```diff
@@ -140,7 +144,9 @@ class PlayerManager: NSObject {
                 //set speed for player
                 audioplayer.rate = self.getSpeed()
 
-                NotificationCenter.default.post(name: Notification.Name.AudiobookPlayer.bookReady, object: nil)
+                DispatchQueue.main.async {
```

Comment:
This dispatch shouldn't be necessary, it's already inside a dispatch main async 🤔